components/Badge/Badge.css

```diff
@@ -0,0 +1,67 @@
+@import url(/Asimov-UI/style.css);
+@import url(/Asimov-UI/components/Avatar/Avatar.css);
+.badge-wrapper {
+  width: fit-content;
+  position: relative;
+
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+}
+
+.badge {
+  position: absolute;
+
+  border-radius: 50%;
+  font-weight: bold;
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+}
+
+.badge-top {
+  top: -11px;
+  right: -13px;
+}
```

Comment:
i guess giving absolute value will not work for every cases
if its working then alright 👍

Reply:
If text is not multiline it works perfectly if its multiline its  kind of off but okay the main reason for using fixed values was when I used %s I was not able to place the badge at the top it kind of stays on the text even when I use top:1%